Glue documentation review (#1456)
spryker/spryker-docs: docs/sdk/dev/initialize-and-run-workflows.md

> # Workflow SDK tool

Author: ```suggestion
---
title: Initialize and run workflows
description: Learn about the Spryker SDK telemetry configuration. 
template: howto-guide-template
redirect_from:
    - /docs/scos/dev/sdk/initializing-and-running-workflows.html
---
# Workflow SDK tool
```

## How to initialize and run workflows

To initialize a project with a specific workflow, execute the following command:

```bash
spryker-sdk init:sdk:project --workflow={workflowName}
```

> If the project already has been initialized with the `spryker-sdk init:sdk:project` command, you can manually add workflow to the project config file `{projectDir}/.ssdk/settings`:

Author: ```suggestion
If the project has already been initialized with the `spryker-sdk init:sdk:project` command, you can manually add workflow to the project config file `{projectDir}/.ssdk/settings`:
```

```

---

> {% info_block warningBox "Project Limits" %}

Author: ```suggestion
{% info_block warningBox Project Limits %}
```

```

---

> **Note:** Project is limited to the workflows specified during initialization.

Author: {% info_block warningBox "Project Limits" %}

The project is limited to the workflows specified during initialization.

 {% endinfo_block %}

> **Note:** Project is limited to the workflows specified during initialization.

Author: ```suggestion
{% info_block warningBox "Project Limits" %}

The project is limited to the workflows specified during initialization.

{% endinfo_block %}```

You can start any workflow by providing its name in the `sdk:workflow:run` command.

---

> **Note:** Two identical top-level workflows can't run inside the same project.

Author: {% info_block warningBox "Identical workflow exclusion" %}

Two identical top-level workflows can't run inside the same project.

 {% endinfo_block %}

> **Note:** Two identical top-level workflows can't run inside the same project.

Author: ```suggestion
{% info_block warningBox "Identical workflow exclusion" %}

Two identical top-level workflows can't run inside the same project.

{% endinfo_block %}```

You can start any workflow by providing its name in the `sdk:workflow:run` command.

---

> {% info_block warningBox "Identical workflow exclusion" %}

Reviewer: Please remove ``` from the end here and in other places where you have. The notes might not be built properly with them

> {% info_block warningBox "Identical workflow exclusion" %}

Author: ```suggestion
{% info_block warningBox Identical workflow exclusion %}
```

  ```bash
  spryker-sdk sdk:workflow:list
  ```

> - To generate and .SVG image for a specific workflow:

Author: ```suggestion
- To generate an .SVG image for a specific workflow:
```

  spryker-sdk sdk:init:project --workflow={workflowName} --workflow={workflowName}
  ```
  ---

>   **Note:** If you initialize workflows for the project, you can use only these workflows.

Author: {% info_block warningBox "Workflow Initialization" %}

If you initialize workflows for the project, you can use only these workflows.

 {% endinfo_block %}

>   **Note:** If you initialize workflows for the project, you can use only these workflows.

Author: ```suggestion
{% info_block warningBox "Workflow Initialization" %}

If you initialize workflows for the project, you can use only these workflows.

{% endinfo_block %}
```

## Configuration reference

> In the SDK directory, the workflows are defined in the workflow YAML files either in `config/packages`, or in the configuration of extension bundles.

Author: ```suggestion
In the SDK directory, the workflows are defined in the workflow YAML files, located  either in `config/packages`, or in the configuration of extension bundles.
```

  spryker-sdk sdk:init:project --workflow={workflowName} --workflow={workflowName}
  ```
  ---

> {% info_block warningBox "Workflow Initialization" %}

Author: ```suggestion
{% info_block warningBox Workflow Initialization %}
```

## Configuration reference

In the SDK directory, the workflows are defined in the workflow YAML files, located  either in `config/packages`, or in the configuration of extension bundles.

See the basic workflow documentation in the [Symfony docs](https://symfony.com/doc/current/workflow.html).

Additionally, you can configure and extend the workflow behavior by providing some specific metadata options:

> - `transitionResolver`: `sdk:workflow:run`. The `service` should implement `\SprykerSdk\SdkContracts\Workflow\TransitionResolverInterface` as shown in the [example workflow definition](example-workflow-definition).

Author: ```suggestion
- `transitionResolver`: `sdk:workflow:run`. The `service` should implement `\SprykerSdk\SdkContracts\Workflow\TransitionResolverInterface` as shown in the [example workflow definition](#example-workflow-definition).
```
